src/views/ResultsContainer.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <section>
+    <button @click="setSelectedComponent('results-list')">List View</button>
+    <button @click="setSelectedComponent('results-map')">Map View</button>
+    <keep-alive>
+      <component 
+        :is="selectedComponent" 
+        :searchResults="searchResults"></component>
+    </keep-alive>
+  </section>
+</template>
+
+<script>
+import ResultsList from '../components/ResultsList.vue'
+import ResultsMap from '../components/ResultsMap.vue'
+export default {
+  data() {
+    return {
+      selectedComponent: "results-list",
+      searchResults: [
```

Comment:
When we connect with the back-end team, we'll have to substitute this hard-coded `searchResults` array for the array we receive from our GET request to their custom API.  Just a note for later.  This should be the format of the array as well, so hopefully we won't run into any issues when it comes to using the `searchResults` array in our `ResultsMap` component.

Reply:
Definitely!